cypress/integration/a11y.test.js

```diff
@@ -0,0 +1,29 @@
+describe('Accessibility tests', () => {
+  beforeEach(() => {
+    cy.intercept('/graphql/?operationName=BrowserProductQuery*').as(
+      'productQuery'
+    )
+
+    cy.intercept('/graphql/?operationName=GalleryQuery*').as('galleryQuery')
+  })
+  it('Test Home', () => {
```

Comment:
You don't need to say it's a test, it's implicit. I think it's better to just leave `Home`. The same in the next tests.

Reply:
You're right. Thank you for the comment